config/app/init/initializers/workers.json

```diff
@@ -0,0 +1,18 @@
+{
+    "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^4.0.0/components/context.jsonld",
+    "@graph": [
+      {
+        "comment": "Spawns the required amount of workers",
+        "@id": "urn:solid-server:default:WorkerManager",
+        "@type": "WorkerManager",
+        "clusterManager": { "@id": "urn:solid-server:default:ClusterManager" }
+      },
+      {
+        "comment": "Cluster manager settings",
+        "@id": "urn:solid-server:default:ClusterManager",
+        "@type": "ClusterManager",
+        "workers": { "@id": "urn:solid-server:default:variable:workers" }
+      }
+    ]
+  }
+  
```

Comment:
Not exactly sure what is going on here but something is wrong about this newline.